catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/CatalogCommands.java

```diff
@@ -115,28 +115,6 @@ protected CatalogFacade getCatalog() throws InterruptedException {
                 .findFirst();
     }
 
-    protected StringBuilder getUserInputModifiable() throws IOException {
```

Comment:
👍  Kind of glad to see this gone

Reply:
![giphy](https://user-images.githubusercontent.com/9013416/30459561-61c2c500-9966-11e7-9375-c95ec40dbd52.gif)